### ISSUE TITLE
Unable to search report

### ISSUE THREAD

**Issue by kaihendry** [GoogleChrome/lighthouse#11937]
I'm unable to ctrl+f search the report in Chromium 87.0.4280.141

https://s.natalian.org/2021-01-11/cannot-search.mp4

My workaround is to have the report as HTML `email.thealaskaguys.com-20210111T144312.html`, re-open it, and then ctrl+f. However the next problem is a lot of text is hidden under a usability nightmare of accordions which I don't know how to easily expand all.

Sidenote: Documentation issue: No audits tab: https://developers.google.com/web/tools/lighthouse#devtools

**Comment by connorjclark** [comment]
Yeah, that's frustrating. I started a discussion with some Chrome DevTools folk about how we could accomplish this.

> Sidenote: Documentation issue: No audits tab: developers.google.com/web/tools/lighthouse#devtools

Used to be called the Audits panel. I'll hunt for more stale references and edit.
